defaults/main.yml

```diff
@@ -1,34 +1,64 @@
 ---
 ## General
-blackbox_exporter_version: 0.4.0
+# Version
+blackbox_exporter_version: 0.18.0
 blackbox_exporter_release_system: linux-amd64
 
+# Package URL
+blackbox_exporter_release_name: "blackbox_exporter-{{ blackbox_exporter_version }}.{{ blackbox_exporter_release_system }}"
+blackbox_exporter_package: "{{ blackbox_exporter_release_name }}.tar.gz"
+blackbox_exporter_url: "https://github.com/prometheus/blackbox_exporter/releases/download/v{{ blackbox_exporter_version }}/{{ blackbox_exporter_package }}"
```

Comment:
I think variables like this one should be moved to /vars, as they are not intended to be seen/used by the role user.

Reply:
I don't agree with this. If you move to vars these variables and i.e. the url changes, there are no simple options for overriding the value :-) (because variable precedence tree)

Reply:
In fact I ran into that issue this morning with zookeeper role :laughing: but I fear we are offering too much default vars (not  just that one in particular)

Reply:
Default vars can be distributed in several files :-)

Reply:
I don't know about other variables, but at least for package related stuff it should be user overridable in case the package sources changes, I'm thinking about cases like [jenkins_role where archives and mirrors tend to change suddenly](https://github.com/idealista/jenkins_role/commit/5465d426b289c7fbcd2a212aa91cb6d1fc001a26) .
Role variables should be values that hardly change, for instance path to role templates.